extension/wallet/src/lib.rs

```diff
@@ -225,10 +225,22 @@ pub async fn make_sell_create_swap_payload(
 pub async fn make_loan_request(
     wallet_name: String,
     collateral: String,
+    fee_rate: String,
+    timeout: String,
 ) -> Result<JsValue, JsValue> {
-    let collateral = map_err_from_anyhow!(parse_to_bitcoin_amount(collateral))?;
+    // TODO: Change the UI to handle SATs not BTC
```

Comment:
I agree with this. Shall we record a ticket for this?

Reply:
https://github.com/comit-network/waves/issues/205